rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -234,20 +379,129 @@ struct BoundingBoxBatch: public MetaDataBatch
     {
         return std::make_shared<BoundingBoxBatch>(*this);
     }
+    void copy_data(std::vector<void*> buffer) override
+    {
+        if(buffer.size() < 2)
+            THROW("The buffers are insufficient") // TODO -change
+        int *labels_buffer = (int *)buffer[0];
+        double *bbox_buffer = (double *)buffer[1];
```

Comment:
Change this to float here, since open source has only float data type 
```suggestion
        double *bbox_buffer = (double *)buffer[1];
```

Reply:
Changed to double